mvp-spec/json-schemas/transmission-response-no-recursion.json

```diff
@@ -0,0 +1,59 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Transmission Response",
+    "description": "",
+    "$id": "transmission-response-no-recursion.json",
+    "type": "object",
+    "properties": {
+        "version": {
+            "$ref": "version.json"
+        },
+        "receiver": {
+            "$ref": "transmission-receiver.json"
+        },
+        "contents": {
+            "type": "array",
+            "items": {
+                "description": "An association of Content-Id with a Transaction Id for building the Audit Log later",
+                "properties": {
+                    "transaction_id": {
+                        "$ref": "transaction_id.json"
+                    },
+                    "content_id": {
+                        "$ref": "transmission-content_id.json"
+                    } 
+                }
+            }
+        },
+        "status": {
+            "$ref": "transmission-status.json",
+            "description": "Equals \"success\". Transmission Responses with a different status from Suppliers must be dismissed.",
+            "enum": [
+                "success"
+            ]
+        },
+        "details": {
+            "$ref": "transmission-details.json"
+        },
+        "source": {
+            "description": "Signature based on input:\n```\ntransmission_response.receiver                + '\\u2063' +\ntransmission_response.status                  + '\\u2063'\ntransmission_response.source.domain           + '\\u2063' +\ntransmission_response.source.timestamp        + '\\u2063' +\nseed.source.signature+ '\\u2063' +\ncontents[0].transaction_ids + '\\u2063' +\ncontents[0].content_id + '\\u2063' +\n... + '\\u2063' +\ncontents[n].transaction_ids + '\\u2063' +\ncontents[n].content_id\n```",
```

Comment:
You might remove `transmission_response.` prefix for clarity and consistency with other signature inputs.